control/veda_file_discovery.py

```diff
@@ -125,162 +127,199 @@ def about_video_validate(self, meta, key):
             # have associated 3rd party transcription provider API keys.
-            process_transcription = False
+            transcript_preferences = None
         except ValueError:
-            LOGGER.error('[VIDEO-PIPELINE] File Discovery - Invalid transcripts preferences=%s', transcript_preferences)
-            process_transcription = False
+            LOGGER.exception('[File Discovery] Invalid transcripts preferences=%s', transcript_preferences)
+            transcript_preferences = None
 
-        # Trigger Ingest Process
-        video_metadata = dict(
-            s3_filename=edx_filename,
-            client_title=client_title,
-            file_extension=file_extension,
-            platform_course_url=course_url,
-        )
-        if process_transcription:
-            video_metadata.update({
-                'process_transcription': process_transcription,
-                'provider': transcript_preferences.get('provider'),
-                'three_play_turnaround': transcript_preferences.get('three_play_turnaround'),
-                'cielo24_turnaround': transcript_preferences.get('cielo24_turnaround'),
-                'cielo24_fidelity': transcript_preferences.get('cielo24_fidelity'),
-                'preferred_languages': transcript_preferences.get('preferred_languages'),
-                'source_language': transcript_preferences.get('video_source_language'),
-            })
-
-        ingest = VedaIngest(
-            course_object=course_query[0],
-            video_proto=VideoProto(**video_metadata),
-            node_work_directory=self.node_work_directory
-        )
-        ingest.insert()
-
-        if ingest.complete is False:
-            return
+        return transcript_preferences
 
+    def discover_studio_ingested_videos(self):
         """
-        Delete Original After Copy
+        Discovers studio ingested videos, for further validations and processes.
         """
-        if self.auth_dict['edx_s3_ingest_prefix'] is not None:
-            new_key = 'prod-edx/processed/' + key.name[::-1].split('/')[0][::-1]
-            key.copy(self.bucket, new_key)
+        if self.node_work_directory:
+            try:
+                connection = boto.connect_s3()
+                self.bucket = connection.get_bucket(self.auth_dict['edx_s3_ingest_bucket'])
+                for video_s3_key in self.bucket.list(self.auth_dict['edx_s3_ingest_prefix'], '/'):
+                    self.validate_and_feed_to_ingest(video_s3_key=self.bucket.get_key(video_s3_key.name))
+            except S3ResponseError:
+                ErrorObject.print_error(message='[File Ingest] S3 Ingest Connection Failure')
+        else:
+            ErrorObject.print_error(message='[File Ingest] No Working Node directory')
 
-        key.delete()
+    def validate_and_feed_to_ingest(self, video_s3_key):
+        """
+        Validates the video key and feed it to ingestion phase.
```

Comment:
Can we rename 'validate' to something else? Usually 'validate' refers to the video file's corruptibility in this repo.